test/EndToEnd/ProjectTemplates/UwpPackageRefClassLibrary.zip/ClassLibrary47.csproj

```diff
@@ -11,8 +11,8 @@
     <AssemblyName>$safeprojectname$</AssemblyName>
     <DefaultLanguage>en-US</DefaultLanguage>
     <TargetPlatformIdentifier>UAP</TargetPlatformIdentifier>
-    <TargetPlatformVersion Condition=" '$(TargetPlatformVersion)' == '' ">10.0.18362.0</TargetPlatformVersion>
-    <TargetPlatformMinVersion>10.0.10586.0</TargetPlatformMinVersion>
+    <TargetPlatformVersion Condition=" '$(TargetPlatformVersion)' == '' ">10.0.19041.0</TargetPlatformVersion>
+    <TargetPlatformMinVersion>10.0.17763.0</TargetPlatformMinVersion>
```

Comment:
I didn't look into the CI failures, but be careful about changing these values. One of the two values determines which version of the Windows SDK VS will look for on the machine, and if that version of the Windows SDK isn't defined in the vsconfig file that the DartLab machine uses for configuration, VS will pop up a dialog that will freeze the test until the test, or the entire pipeline, times out.

Reply:
Good catch! Thanks! Fixed.